LibTest/io/Stdout/Stdout_class_A02_t01.dart

```diff
@@ -20,14 +20,14 @@ Stream<List<int>> stream1 = new Stream<List<int>>.fromIterable(
 Stream<List<int>> stream2 = new Stream<List<int>>.fromIterable(
     [[1, 2, 3, 4, 5], [12], [3, 22]]);
 
-test(Stdout sink, Stream<List<int>> stream) async {
+test(Stdout sink, Stream<List<int>> stream) {
   sink.addStream(stream).then((x) {
     new Future.delayed(new Duration(seconds: 3));
```

Comment:
What is the purpose of this line here? The test isn't using `asyncStart()` so basically after `main` returns we expect the test to be successfully done, so any delayed actions are ignored.

I think a slightly better test would be
```dart
Future test(Stdout sink) async {
  final future = sink.addStream(Stream.fromIterable([[1, 2, 3, ]));

  // While the `sink.addStream()` is in-progress we cannot touch the sink.
  Expect.throws(() { sink.writeln(); }, (e) => e is StateError);
  // After the `sink.addStream()` is done, we can write to it again.
  await future;
  sink.writeln();
}
main() async {
  asyncStart();
  await test(stdout);
  await test(stderr);
  asyncEnd();
}
```